frontend/src/app/sessions/active-sessions/upload-dialog/upload-dialog.component.html

```diff
@@ -29,6 +29,15 @@ <h1>Upload Files (Experimental)</h1>
   </button>
```

Comment:
The title of the dialog should be `File browser (experimental)` instead of `Upload Files (Experimental)`.

Reply:
done.